cli/src/services/config.ts

```diff
@@ -71,4 +84,17 @@ Only your master language should be listed in your files config.`
   private sourceFolderPath(source: string) {
     return source.replace(path.basename(source), '');
   }
+
+  private getCurrentBranchName() {
+    return execSync('git rev-parse --abbrev-ref HEAD')
```

Comment:
What if `git` is not present? We should catch all errors to return an empty string